acclimatise/model.py

```diff
@@ -388,18 +388,16 @@ def capital(self):
 int_num_re = re.compile(r"([+-]?[0-9]+)", flags=re.IGNORECASE)
 
 
-def distinguish_inout(string, cls) -> cli_types.CliFileSystemType:
+def distinguish_inout(string, cls) -> typing.Type[cli_types.CliFileSystemType]:
```

Comment:
No, what I meant was that `def distinguish_inout(string: str, cls: typing.Type[cli_types.CliFileSystemType]) -> cli_types.CliFileSystemType`

Since cls is a class and the return value is an instance of that class. `typing.Type` indicates a class.

Reply:
OK